### PR TITLE
Add `enableFix` option to `@intlify/vue-i18n/no-unused-keys` rule

### LINKED ISSUE #5: support `eslint --fix`
we need the custom parser of JSON for ESLint.

### PR DESCRIPTION
**Pull request by ota-meshi**
This PR adds quick fix to remove unused keys, and an `enableFix` option to remove unused keys with `--fix`, to the `@intlify/vue-i18n/no-unused-keys` rule.

The quick fix is available in editors that support the Suggestions API. e.g. VS Code
https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions

Close #5.

~I think we first need to support hot-reloading to add autofix to `@intlify/vue-i18n/no-unused-keys` rule.
If we add autofix without hot-reloading support, I think user may not be able to add new keys due to cache.~
It can be autofix only when the `enableFix` option is `true`. Users should not enable the `enableFix` option if they want to avoid caching issues.

---

In addition, the following changes are included.

- Change to use eslint-plugin-jsonc.
- Remove json processor as it is no longer needed.
- Add `/base` config.

### CODE REVIEW

**Comment by kazupon** [comment]
Thank you for your PR!
That's great works!
LGTM! 👍 

> If we add autofix without hot-reloading support, I think user may not be able to add new keys due to cache.

Yeah!
You're right.
I think that we should be released autofix as experimental feature.

**Comment by ota-meshi** [comment]
@kazupon Thank you for your reply.

The `no-unused-keys` rule does not yet support autofix (eslint --fix) in this PR. The `no-unused-keys` rule only supports quick fix.
Do you want to add an option (e.g. `enableFix`) to the `no-unused-keys` rule to support autofix?

**Comment by kazupon** [comment]
@ota-meshi 
> Do you want to add an option (e.g. `enableFix`) to the `no-unused-keys` rule to support autofix?

I hope it! 😉

**Comment by ota-meshi** [comment]
@kazupon I changed this PR to add the `enableFix` option!

File: docs/rules/README.md [hunk 1]
```diff
@@ -17,5 +17,5 @@
 | Rule ID | Description |    |
 |:--------|:------------|:---|
 | [@intlify/vue-i18n/<wbr>no-dynamic-keys](./no-dynamic-keys.html) | disallow localization dynamic keys at localization methods |  |
-| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys |  |
+| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys | :black_nib: |
```
Comment (kazupon) [inline docs/rules/README.md:20]:
```suggestion
| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys | :black_nib: **(experimental)** |
```

Comment (kazupon) [reply docs/rules/README.md:20]: @ota-meshi 
I've added `experimental`  word to here.

Comment (ota-meshi) [reply docs/rules/README.md:20]: @kazupon 
This list is generated by the `generate` script, how would you like to add it?

- Add the `experimentalFixable` property to `mea.docs` of `no-unused-keys` rule.
- Change the `update-docs-index` to add words only for the `no-unused-keys` rule.
- Other?

Comment (kazupon) [reply docs/rules/README.md:20]: @ota-meshi 
Ohh, I forgot it...
Don't worry about it. 😅

**Comment by kazupon** [comment]
@ota-meshi 
You can merge this PR! 
If you are possible, please, you will try to be publishing to npm. 🚀

**Comment by ota-meshi** [comment]
@kazupon Thank you for checking this PR. I'll publish to npm later.

**Pull request merged**